### ISSUE TITLE
Create sample content for an application including WCAG

### ISSUE THREAD

**Issue by grugnog** [GSA/open-product-accessibility-template#23]
We can start from [this Drupal VPAT](https://github.com/CivicActions/accessibility/blob/main/VPAT/Drupal9AccessibilityConformanceReport.md) that is already in a Markdown format.

**Comment by mgifford** [comment]
I think with this PR we should be able to close this issue https://github.com/GSA/open-product-accessibility-template/pull/84
